workflow/rules/merge_process.smk

```diff
@@ -7,13 +7,17 @@ rule fast_alignment:
         "../envs/merge_process.yml"
     input:
         # metadata files with image locations
-        PREPROCESS_OUTPUTS["combine_metadata_phenotype"],
+        lambda wildcards: output_to_input(
+            PREPROCESS_OUTPUTS["combine_metadata_phenotype"],
+            {} if config["merge_process"]["ph_metadata_channel"] is None else {"channel": config["merge_process"]["ph_metadata_channel"]},
+            wildcards,
+        ),
         lambda wildcards: output_to_input(
             PREPROCESS_OUTPUTS["combine_metadata_sbs"],
-            {"cycle": config["merge_process"]["sbs_metdata_cycle"]},
+            {"cycle": config["merge_process"]["sbs_metadata_cycle"], 
+             **({} if config["merge_process"]["sbs_metadata_channel"] is None else {"channel": config["merge_process"]["sbs_metadata_channel"]})},
```

Comment:
Are these changes specific to well-based processing? I'm having some trouble understanding what's happening here... maybe we can discuss in person?

If we don't expect this approach to be used by most people processing OPS data than better to leave it out and keep things simple imo.